src/python/analyzer_executor/src/analyzer_executor_lib/analyzer_executor.py

```diff
@@ -444,10 +444,13 @@ def emit_event(s3: S3ServiceResource, event: ExecutionHit, is_local: bool) -> No
     # TODO fargate: always emit manual events
     if is_local:
         # Local = manual eventing
+
+        deployment_name = os.environ["DEPLOYMENT_NAME"]
```

Comment:
might be a candidate for grapl_common `env_helpers.py` perhaps

Reply:
Good idea... I'll file an issue for this as well.

Reply:
https://github.com/grapl-security/issue-tracker/issues/334